src/commonMain/kotlin/io/github/koalaplot/core/util/Gestures.kt

```diff
@@ -0,0 +1,220 @@
+package io.github.koalaplot.core.util
+
+import androidx.compose.foundation.gestures.awaitEachGesture
+import androidx.compose.foundation.gestures.awaitFirstDown
+import androidx.compose.foundation.gestures.calculateCentroid
+import androidx.compose.foundation.gestures.calculateCentroidSize
+import androidx.compose.foundation.gestures.calculatePan
+import androidx.compose.foundation.gestures.calculateZoom
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.pointer.PointerEvent
+import androidx.compose.ui.input.pointer.PointerInputScope
+import androidx.compose.ui.input.pointer.positionChanged
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.util.fastAny
+import androidx.compose.ui.util.fastForEach
+import kotlin.math.abs
+
+internal const val ZoomZeroPoint = 1.0f
```

Comment:
It's a bit confusing using a name with Zero when the value is one. A couple of alternatives are ZoomNeutralPoint or UnitZoomPoint.

Reply:
Moved this to ZoomFactor, and named the constant as NeutralPoint